frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -264,16 +270,15 @@ const CreateVersionModal = function CreateVersionModal({
         </div>
       </div>
 
-      <div className="mb-3">
+      <div className="mb-3" style={{ padding: '2rem 0' }}>
         <label className="form-label">Create version from</label>
         <div className="ts-control">
-          <select className="form-select">
-            {appVersions.map((version) => (
-              <option className="dropdown-item" key={version.id} onClick={() => setCreateAppVersionFrom(version)}>
-                {version.name}
-              </option>
-            ))}
-          </select>
+          <Select
+            options={options}
+            maxMenuHeight={100}
+            menuPlacement="auto"
+            onChange={(version) => setCreateAppVersionFrom(version)}
+          />
```

Comment:
How about adding a defaultValue prop here?
```suggestion
          <Select
            options={options}
            maxMenuHeight={100}
            menuPlacement="auto"
            defaultValue={options[options.length - 1]}
            onChange={(version) => setCreateAppVersionFrom(version)}
          />
```

Reply:
Yes, presently, even though the default selected version is `v1`, it will create version from the editing version instead of `v1`, unless we drop it down and click on it. @manishkushare